bin.js

```diff
@@ -31,3 +34,12 @@ function getConfig() {
   }
 }
 
+function getRules(set) {
+  var configFileName = set ? ('eslint-config-' + set) : null
```

Comment:
I investigated this a bit and came to the conclusion that we could use eslint itself to resolve all this for us.
That is, we could `require('./node_modules/eslint/lib/config.js')`, instantiate the exposed `Config` object and simply use its `getConfig` method, it will return an object with a property `rules` — from there on, it would be the same procedure that is already implemented.

**But ...**
- We'd have to make eslint a (peer) dependency, which (in my opinion) would be just fine, as the whole purpose of this module is to work with it.
- We'd have to require a part of eslint's internals that are not part of their public api, when used as a lib (as in `require('eslint')`).

Only drawback I can think of would be breaking when they change their internal architecture, concerning this certain part.